x-pack/plugins/canvas/public/components/fullscreen_control/fullscreen_control.js

```diff
@@ -9,6 +9,12 @@ import PropTypes from 'prop-types';
 import { Shortcuts } from 'react-shortcuts';
 
 export class FullscreenControl extends React.PureComponent {
+  keyHandler = action => {
+    if (action === 'FULLSCREEN' || (this.props.isFullscreen && action === 'FULLSCREEN_EXIT')) {
```

Comment:
nit: can we make these `const`s?

Reply:
I don't understand what you're asking for here.